src/directives/if-defined.ts

```diff
@@ -27,6 +27,12 @@ export const ifDefined = directive((value: unknown) => (part: Part) => {
       part.committer.element.removeAttribute(name);
     }
   } else {
+    if (part instanceof AttributePart &&
```

Comment:
Wouldn't property parts need protection, too?

Reply:
Right... though this directive doesn't do anything interesting on any part other than attribute parts. They all need a fix, and I maybe regret making this work in those bindings. Also, the line below is wrong and I need a better test. This is definitely a flaw in the directive hook implementation though :( Need to think about this some more.

Reply:
I added a test for node parts which will also apply to other part types.